asv/util.py

```diff
@@ -81,28 +81,28 @@ def reraise(self):
             raise self
 
 
-def human_list(l):
+def human_list(non_human_list):
```

Comment:
This is a normal list that wants to be printed in a human readable way. `non_human_list` doesn't seem to be a very clear name of what it is, maybe better just `input` or `input_list` or even `list_to_format`.